spring/src/test/java/org/axonframework/spring/serialization/avro/test1/ComplexObject.java

```diff
@@ -0,0 +1,477 @@
+/**
```

Comment:
Same class-level JavaDoc and copyright question as with the previous `ComplexObject`.